spec/cucumber/deprecate_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Cucumber::Deprecate
+  describe CliOption do
+    let(:subject) { Cucumber::Deprecate::CliOption }
```

Comment:
If you want to, you can use `described_class` instead of defining a `subject` with the same value.

Reply:
nice, thanks for the tip :)

I'm often confused with `subject {...}`, `let(:subject) { ...}` or not defining any subject at all to be honest :D

Reply:
Glad it helped!